indico/modules/attachments/models/folders.py

```diff
@@ -90,6 +90,13 @@ def __table_args__(cls):
         nullable=False,
         default=True
     )
+    #: If the folder is never shown in the frontend (even if you can access it)
+    is_hidden = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+
```

Comment:
one linebreak too much (there should be one empty line between this and the the relationships)

Reply:
Done